src/sentry/models/outbox.py

```diff
@@ -53,6 +53,7 @@ class OutboxScope(IntEnum):
     INTEGRATION_SCOPE = 5
     APP_SCOPE = 6
     TEAM_SCOPE = 7
+    PROVISION_SCOPE = 8
```

Comment:
If organization provisioning is a separate scope could we end up processing organization scope messages before provision scopes?

Reply:
Provisioning scope is used as the first step, before the org should exist.  I considered re-using USER scope here, but it seemed much more clear to have a scope specifically for provisioning.  In general, provisioning has no other ordering requirements.  After an org is created, the secondary provisioning (subscription) also runs in the provisioning scope, just to simplify the flush action (which waits for all provisioning of the user to have completed) in the controller.

Reply:
I re-read the getsentry changes and the scenario I was thinking of doesn't appear to be possible, as the provisioning scope outbox message is processed well before organization creation is done.